mender-convert-functions.sh

```diff
@@ -738,9 +738,6 @@ set_fstab() {
 
 extract_file_from_image() {
   local cmd="dd if=$1 of=${output_dir}/$4 skip=$2 bs=512 count=$3"
-
-  [ "${4##*.}" == "ext4" ] && { log "\tStoring data in $4 root file system image."; } \
-                           || { log "\tStoring data in $4 disk image."; }
```

Comment:
It was @estenberg expectation that during Mender Artifact creation we will inform user about root filesystem being dumped. Especially that this image is further used during tests.
I agree that we shouldn't assume that it is always ext4, hence I propose to check the filesystem type of desired partition, e.g. 
`parted -m <MENDER_IMAGE> print | tail -n +3 | awk -F ":" '{ if(NR==1/2) print $(NF-2)}'`
where NR reflects primary or secondary partition selected as the base for an artifact.
Above command can be issued in `do_mender_disk_image_to_artifact()` method before root filesystem is to be extracted.

Reply:
That part stays the same, this is just an early print out that is effected. Which to me does not add much value, but adds "complexity"

Going from:

```
1/9 Repartitioning raw disk image...
	Detected raw disk image with 2 partition(s).
	Creating Mender disk image:       
		image size: 3825205248 bytes       
		boot partition size: 46137344 bytes       
		root filesystem size: 1816133632 bytes       
		data partition size: 134217728 bytes
	Changes in partition table applied.
2/9 Formatting repartitioned raw disk image...
	Creating MS-DOS filesystem for 'boot' partition.
	Creating ext4 filesystem for 'primary' partition.
	Creating ext4 filesystem for 'secondary' partition.
	Creating ext4 filesystem for 'data' partition.
3/9 Setting boot partition...
	Extracting boot partition from raw disk image.
	Storing data in boot.vfat disk image.
	Done.
4/9 Setting root filesystem partition...
	Extracting root filesystem partition from raw disk image.
	Storing data in rootfs.img disk image.
	Done.
5/9 Setting file system table...
	Done.
6/9 Cleaning intermediate files...
	Partition mappings cleaned.
7/9 Installing Mender to Mender disk image...
	Downloading inventory & identity scripts.
	Installing files.
	Done.
8/9 Installing Bootloader to Mender disk image...
	Building U-Boot related files.
	Installing U-Boot related files.
	Done.
9/9 Creating Mender Artifact...
	Rootfs partition id not set - 'primary' will be used by default.
	Storing data in 2018-11-13-raspbian-stretch-lite.ext4 root file system image.
	Writing Mender artifact to: /mender-convert/output/2018-11-13-raspbian-stretch-lite.mender
	Creating Mender Artifact succeeded.
Conversion complete!
The Mender disk image you can provision your device storage with is at:         
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.sdimg
The Mender root file system partition is at:
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.ext4
The Mender Artifact you can upload to your Mender server to deploy to your devices is at:         
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.mender
``` 

to

```
1/9 Repartitioning raw disk image...
	Detected raw disk image with 2 partition(s).
	Creating Mender disk image:       
		image size: 3825205248 bytes       
		boot partition size: 46137344 bytes       
		root filesystem size: 1816133632 bytes       
		data partition size: 134217728 bytes
	Changes in partition table applied.
2/9 Formatting repartitioned raw disk image...
	Creating MS-DOS filesystem for 'boot' partition.
	Creating ext4 filesystem for 'primary' partition.
	Creating ext4 filesystem for 'secondary' partition.
	Creating ext4 filesystem for 'data' partition.
3/9 Setting boot partition...
	Extracting boot partition from raw disk image.
	Done.
4/9 Setting root filesystem partition...
	Extracting root filesystem partition from raw disk image.
	Done.
5/9 Setting file system table...
	Done.
6/9 Cleaning intermediate files...
	Partition mappings cleaned.
7/9 Installing Mender to Mender disk image...
	Downloading inventory & identity scripts.
	Installing files.
	Done.
8/9 Installing Bootloader to Mender disk image...
	Building U-Boot related files.
	Installing U-Boot related files.
	Done.
9/9 Creating Mender Artifact...
	Rootfs partition id not set - 'primary' will be used by default.
	Writing Mender artifact to: /mender-convert/output/2018-11-13-raspbian-stretch-lite.mender
	Creating Mender Artifact succeeded.
Conversion complete!
The Mender disk image you can provision your device storage with is at:         
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.sdimg
The Mender root file system partition is at:
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.ext4
The Mender Artifact you can upload to your Mender server to deploy to your devices is at:         
	/mender-convert/output/2018-11-13-raspbian-stretch-lite.mender
```

I simply like removing stuff :), and trying to get rid of special handling which is not necessary.

Or there can be a unconditional logs:

    log "\tStoring data in $4 image."

To hint name of the image that is extracted.

Reply:
Updated the PR, and this commit. It now unconditionally does:

     log "\tStoring data in $4."